src/ASM/ASMs2D.C

```diff
@@ -695,17 +695,17 @@ bool ASMs2D::connectBasis (int edge, ASMs2D& neighbor, int nedge, bool revers,
   for (size_t i = 0; i < masterNodes.size(); i++)
   {
     int node = masterNodes[i];
-    int slave = slaveNodes[revers ? slaveNodes.size()-i-1 : i];
+    int lslave = slaveNodes[revers ? slaveNodes.size()-i-1 : i];
```

Comment:
Then better (more consistent) to rename both variable, to `mnode` (from node) and `snode`, respectively.
Alternatively, just reuse the input variables `master`and `slave`, since done with those values here anyway.
I prefer the first option though.

Reply:
mnode and snode it is.